src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -135,27 +137,40 @@ def __init__(self, protected_points,
                beamwidth=3,
                azimuth_range=(0, 360),
                freq_ranges_mhz=[(3550, 3650)],
-               catb_neighbor_dist=ml.CAT_B_NBRHD_DIST_DEFAULT):
+               catb_neighbor_dist=ml.CAT_B_NBRHD_DIST_DEFAULT,
```

Comment:
Similar to the previous comment, "catb_neighbor_dist" should be set to 200 (km), just like you did in zones._LoadDpaZones().

Reply:
The code in zones is temporary while we waited for having the actual distance provided in the KML file, or some other mean. I received this morning a sheet, so I will include that for now, and will update when finally decided what is the final source.

Note that there a still a few details to be figured out (do we still use HAAT and 300km ,etc..) - we'll know better when the DPA WG1 requirement is finalized with latest development.